### ISSUE TITLE
Erratic live.cache error in compilation after modifying .drab files.

### ISSUE THREAD

**Issue by guidotripaldi** [grych/drab#113]
Frequently, but not always, I got a live.cache error  after modifying `.drab` files. To be able to recompile againg I have to `$ rm  "/my/app/path/_build/dev/lib/drab/priv/drab.live.cache"`.
But I don't know which are the exact boundary conditions to reproduce this erratic error.

```
== Compilation error in file lib/my_app_web/views/test_view.ex ==
** (MatchError) no match of right hand side value: {:error, {:not_a_dets_file, "/my/app/path/_build/dev/lib/drab/priv/drab.live.cache"}}
    lib/drab/live/cache.ex:29: Drab.Live.Cache.start/0
    lib/drab/live/eex_engine.ex:171: Drab.Live.EExEngine.init/1
    (eex) lib/eex/compiler.ex:28: EEx.Compiler.compile/2
    (phoenix) lib/phoenix/template.ex:378: Phoenix.Template.compile/2
    (phoenix) lib/phoenix/template.ex:186: anonymous fn/3 in Phoenix.Template."MACRO-__before_compile__"/2
    (elixir) lib/enum.ex:1899: Enum."-reduce/3-lists^foldl/2-0-"/3
    (phoenix) expanding macro: Phoenix.Template.__before_compile__/1
    lib/frontend_web/views/cart_view.ex:1: FrontendWeb.CartView (module)
```

**Comment by grych** [comment]
Yes, deleting the file is a good approach.
DETS cache is a temporary and is going to be replaces by the module before 1.0.

**Comment by grych** [comment]
Fixed in #115, will be included in 0.7.8
